### ISSUE TITLE
Broken link in README

### ISSUE THREAD

**Issue by treigerm** [TuringLang/AdvancedHMC.jl#256]
The link in the second sentence of the parallel sampling sentence points to a webpage which does not seem to exist (https://turing.ml/dev/docs/library/advancedhmc/parallel_sampling). Is this a page that is supposed to be added for the standalone docs?

**Comment by xukai92** [comment]
That might be updated to https://turing.ml/dev/docs/using-turing/guide#sampling-multiple-chains.
But this link is no longer informative to AHMC users, as many of the patterns have been built into AbstractMCMC now and users doesn't need to write from scratch any more if using Turing directly.

**Comment by treigerm** [comment]
The link is supposed to give information about vectorized sampling as the full sentence says:
> It also supports vectorized sampling for static HMC and has been discussed in more detail in the documentation [here](https://turing.ml/dev/docs/library/advancedhmc/parallel_sampling).

Is the vectorized sampling already available for Turing and does AbstractMCMC have an interface for it?

**Comment by xukai92** [comment]
> Is the vectorized sampling already available for Turing and does AbstractMCMC have an interface for it?

No.

IIRC that part was written by @Vaibhavdixit02.
Can you remind us what was that supposed to link to?

**Comment by Vaibhavdixit02** [comment]
Yeah that was supposed to get a separate page in the docs, I think my unfinished PR might have the structure of what it was
